daemon/cmd/main.go

```diff
@@ -454,9 +454,8 @@ var amountSats = cli.UintFlag{
 }
 
 var address = cli.StringFlag{
-	Name:     "address",
-	Usage:    "Address to swap to",
```

Comment:
Please document that as it is optional, if not set LND wallet will be used

Reply:
done